examples/persistent-volume-provisioning/README.md

```diff
@@ -134,6 +135,8 @@ For example:
   'Distribute volume':
     `volumetype: none`
 
+* `volumeoptions` : This option allows you to specify the gluster volume option which has to be set on the provisioned volume. The value string is an abstraction of the real options set on the volume. At present the only available value which can be set by this attribute is `encryption`, if this value has been set, all the required volume options ( for eg. `client.ssl, server.ssl`) for enabling encryption on the volume will be performed by Heketi.
```

Comment:
@msau42  Thanks. At present GlusterFS support almost 200 volume options. So having it passed per parameter is not possible. Most of the time 2/3/4 options will be passed by the administrator. More or less, we are not doing any parsing in glusterfs plugin any more, rather it has been pushed to next layer( heketi). so I think it should be fine for now.